libs/service-portal/petitions/src/screens/PetitionsTable/ExportPetition/downloadCSV.ts

```diff
@@ -0,0 +1,28 @@
+import CSVStringify from 'csv-stringify'
+
+export const downloadCSV = async (
+  name: string,
+  header: string[],
+  data: Array<Array<string | number>>,
+  type: 'csv',
+) => {
+  const getFile = (name: string, output: string | undefined) => {
+    const uri =
+      type === 'csv'
+        ? 'data:text/plain;charset=utf-8,\uFEFF'
+        : 'data:application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;base64,'
```

Comment:
If you don't need the Excel format, the csvStringify function on line 25 could be simplified to:

```
    CSVStringify(data, (err, output) => {
      const encodedUri = encodeURI(`data:text/csv;charset=utf-8,${output}`)
      const link = document.createElement('a')
      link.setAttribute('href', encodedUri)
      link.setAttribute('download', name)
      document.body.appendChild(link)

      link.click()
    })
```